cores/esp32/HWCDC.cpp

```diff
@@ -186,7 +186,6 @@ void HWCDC::begin(unsigned long baud)
         }    
     }
     usb_serial_jtag_ll_disable_intr_mask(USB_SERIAL_JTAG_LL_INTR_MASK);
-    usb_serial_jtag_ll_clr_intsts_mask(USB_SERIAL_JTAG_LL_INTR_MASK);
```

Comment:
I think this should be moved to the end method like it was done in #9331.

Reply:
Do you mean end of the begin method or to the HWCDC::end()?